server/db/models/index.js

```diff
@@ -8,6 +11,31 @@ const Books = require('./books')
  *    BlogPost.belongsTo(User)
  */
 
+User.hasMany(Order, {
+  foreignKey: "user"
+});
+Order.belongsTo(User);
+
+const bookOrders = db.define("book_order", {
+  bookId: {
+    type: Sequelize.INTEGER,
+    references: {
+      model: Book,
+      key: "id"
+    }
+  },
+  orderId: {
+    type: Sequelize.INTEGER,
+    references: {
+      model: Order,
+      key: "id"
+    }
+  }
+});
```

Comment:
Is defining the `bookOrders` model inside `index.js` the way to do it? Just asking because I don't really know how through tables work.

Reply:
Because the model exists for the purpose of facilitating an m:m association, placing it among the project's associations seems natural, and it's what I did for my JPFP. Another, separate file for the `bookOrders` model seems unnecessary to me.

Reply:
Okay, cool